diff_drive_controller/src/speed_limiter.cpp

```diff
@@ -58,7 +58,9 @@ namespace diff_drive_controller
     double min_acceleration,
     double max_acceleration,
     double min_jerk,
-    double max_jerk
+    double max_jerk,
+    double min_deceleration,
```

Comment:
I had them at the end intentionally to keep backwards compatibility of any code that may have invoked the constructor with the old arguments. Having them at the end allows the new params to be default initialized.